packages/treasury/src/vault.js

```diff
@@ -384,7 +384,9 @@ export function makeVaultKit(
 
     const fee = multiplyBy(wantedRun, manager.getLoanFee());
     if (amountMath.isEmpty(fee)) {
-      throw seat.exit('loan requested is too small; cannot accrue interest');
+      throw seat.fail(
```

Comment:
This looks good, however, the `seat` type is currently any. Can we add a `@type {OfferHandler}` to the offerHandler to make sure that the linting checks that the seat methods are called correctly?

Reply:
done (twice)